app/controllers/finders_controller.rb

```diff
@@ -37,4 +38,8 @@ def facet_params
   def keywords
     params[:keywords] unless params[:keywords].blank?
   end
+
+  def atom_url
+    request.original_url.gsub('.json', '').gsub(finder_slug, "#{finder_slug}.atom")
```

Comment:
Could this have a test? It would help show what the intended behaviour is, eg, does this preserve filtering arguments?